src/features/SendAssets/SendAssets.reducer.tsx

```diff
@@ -101,14 +102,14 @@ export const sendAssetsReducer = (state: State, action: ReducerAction): State =>
       return { ...state, send: false, txReceipt };
     }
 
-    case sendAssetsReducer.actionTypes.REQUEST_RESUBMIT: {
+    case sendAssetsReducer.actionTypes.REQUEST_SPEED_UP: {
```

Comment:
Is this used at all?

Reply:
If so, are we using both the reducer and the query params?

Reply:
not used anymore, i used a find&replace to rename :P ill delete tho